client/task-list/tasks/payments/PaymentGatewaySuggestions/components/List/test/list.js

```diff
@@ -0,0 +1,186 @@
+/**
+ * External dependencies
+ */
+import { render } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import { List } from '../List';
+
+jest.mock( '@woocommerce/tracks', () => ( {
+	recordEvent: jest.fn(),
+} ) );
+
+jest.mock( '@wordpress/data', () => {
```

Comment:
Do we need to mock these packages for anything?

Reply:
Looks like I didn't end up using that for any of the remaining tests, so nope!

I removed the `@wordpress/data` mock. I believe I copied the tracks mock from another test, and assumed it was to prevent any junk data from being sent. Unless there's something else in the jest environment that is preventing that?